### PR TITLE
Fix-order-create

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

**Comment by darmhoo** [comment]
I changed the businesstypes slug to what we have on the table.
I applied prettier formatting on the file
I changed the filter option to point to the slug

File: src/app/pages/dashboard/dashboard.page.ts [hunk 1]
```diff
@@ -1,1416 +1,1489 @@
...
+	productType: any;
+	paymentGateways: any;
+	receiptText: boolean = true;
+	salesPlanId: any;
+	myControl = new FormControl();
+	repaymentDuration: any;
+	repaymentCyclesopt: any;
+	downPaymentRates: any;
+	businessTypes: any;
+	busTypeSlug = {
```
Comment (teebarg) [inline src/app/pages/dashboard/dashboard.page.ts]: why do you need this again?

Comment (darmhoo) [reply src/app/pages/dashboard/dashboard.page.ts]: removed

File: src/app/pages/dashboard/dashboard.page.html [hunk 2]
```diff
@@ -337,7 +337,7 @@
                 <mat-form-field>
                     <mat-select matNativeControl placeholder="Business Type" formControlName="business_type_id"
                         required>
-                        <mat-option *ngFor="let plan of businessTypes;" [value]="plan"> {{busTypeSlug[plan.slug] || plan.name}}  </mat-option>
+                        <mat-option *ngFor="let plan of businessTypes;" [value]="plan"> {{plan.name || ''}}  </mat-option>
```
Comment (darmhoo) [inline src/app/pages/dashboard/dashboard.page.html:340]: I fixed it here @teebarg

File: src/app/pages/dashboard/dashboard.page.ts [hunk 3]
```diff
@@ -131,20 +131,7 @@ export class DashboardPage implements OnInit {
...
-		ac_cash_loan: 'Altara Credit Cash Loan',
-		'ap_cash_loan-product': 'Altara Pay Cash Loan (Product)',
-		ap_employee_cash_loan: 'Altara Pay Employee Cash Loan',
-		ap_starter_cash_loan: 'Altara Pay Starter Cash Loan',
-		ap_rentals: 'Altara Pay Rentals',
-		'ap_starter_cash_loan-no_collateral':
-			'Altara Pay Starter Cash Loan(No Collateral)',
-		'ap_cash_loan-no_collateral': 'Altara Pay Cash Loan(No Collateral)',
-	};
+	businessTypes: any;
```
Comment (darmhoo) [inline src/app/pages/dashboard/dashboard.page.ts:134]: and here too @teebarg

File: src/app/pages/dashboard/dashboard.page.ts [hunk 4]
```diff
@@ -1,1416 +1,1489 @@
...
+		let headers = new HttpHeaders({
+			'Content-Type': 'application/json',
+			'Authorization': `Bearer ${localStorage.getItem('token')}`,
+		});
+		let options = { headers: headers };
+		return this.http
+			.get(this.env.NEW_API_URL + '/api/business_type', options)
+			.subscribe((res) => {
+				this.businessTypes = res['data']['data']
+					.filter((data) => data.slug.includes('ap_') && data.status)
```
Comment (darmhoo) [inline src/app/pages/dashboard/dashboard.page.ts:1246]: changed the filter @teebarg

**Review by teebarg — APPROVED** [review]

**Pull request merged**